samples/ironside_se/protectedmem_periphconf/README.rst

```diff
@@ -0,0 +1,173 @@
+
+.. code-block:: console
+
+   [00:00:00.123,456] === Protected Memory Demo ===
+   [00:00:00.123,457] periphconf_partition address: 0x0e040000
+   [00:00:00.123,458] periphconf_partition size: 0x2000 bytes
+   [00:00:00.123,459] Last 16-byte aligned area start: 0x0e041ff0
+   [00:00:00.123,460] Test write address (last 4 bytes): 0x0e041ffc
+   [00:00:00.123,461] Current value at protected address: 0xffffffff
+   [00:00:00.123,462]
+   [00:00:00.123,463] This sample demonstrates UICR.PROTECTEDMEM protection.
+   [00:00:00.123,464] When protected memory is modified, the integrity check will fail
+   [00:00:00.123,465] on the next boot, preventing the device from booting.
+   [00:00:00.123,466]
+   [00:00:00.123,467] Attempting to write test pattern 0xdeadbeef to protected memory...
+   [00:00:00.123,468] Readback value: 0xdeadbeef
+   [00:00:00.123,469] Write successful (in current boot).
+   [00:00:00.123,470]
+   [00:00:00.123,471] Rebooting...
+   [00:00:00.123,472] After reboot, if protection is working correctly:
+   [00:00:00.123,473]   - The integrity check will detect the modification
+   [00:00:00.123,474]   - The device will fail to boot with an integrity error
+   [00:00:00.123,475]   - You should see a boot error status indicating PROTECTEDMEM integrity failure
+   [00:00:00.123,476]
+   [00:00:00.123,477] If protection is NOT working, the device will boot normally
+   [00:00:00.123,478] and you will see this message again.
+
+After the reboot, if protection is working correctly, the secondary firmware will boot instead of the main application.
+You should see the following output from the secondary firmware:
+
+.. code-block:: console
+
+   [00:00:01.234,567] *** Booting nRF Connect SDK v3.1.99-1baadd786027 ***
+   [00:00:01.234,568] *** Using Zephyr OS v4.2.99-d7669eff2b94 ***
+   [00:00:01.234,569] === Secondary Firmware Booted ===
+   [00:00:01.234,570] PROTECTEDMEM integrity check FAILED - device booted secondary firmware
+   [00:00:01.234,571] This indicates that protected memory was modified and integrity check failed.
+
+This demonstrates that the integrity check detected the modification and prevented the main application from booting.
+
+Configuration
+*************
+
+The sample uses the following key configurations:
+
+Device Tree Overlay
+===================
+
+The ``app.overlay`` file relocates the ``periphconf_partition`` to be placed right after ``cpuapp_boot_partition``.
+The ``sysbuild.cmake`` file applies this same overlay to the UICR image so both images see the same partition layout:
+
+.. code-block:: dts
+
+   &mram1x {
+       partitions {
+           periphconf_partition: partition@40000 {
+               reg = <0x40000 DT_SIZE_K(8)>;
+           };
+           cpuapp_slot0_partition: partition@42000 {
+               reg = <0x42000 DT_SIZE_K(328)>;
+           };
+       };
+   };
+
+This places the partition at offset 0x40000 (right after ``cpuapp_boot_partition`` at 0x30000 with size 64KB).
```

Comment:
This information is stated clearly elsewhere, with this amount of duplication maintaining the sample becomes hard if we ever change any of these values.